bin/cordlr

```diff
@@ -2,11 +2,11 @@
 const minimist = require('minimist')
 const Help = require('./methods/help')
 const Version = require('./methods/version')
-const Start = require('./methods/start')
+const Cordlr = require('./methods/start')
```

Comment:
Do you think Start is bad?

How about we call it

`StartMethod`
`VersionMethod`
`HelpMethod`

if you don't like `Start`?